### DISCUSSION TITLE
Supported dimmer with color variations

### DISCUSSION THREAD

**Discussion by lakano** [Koenkk/zigbee2mqtt#5878] [General]
Hello,
For a new house in construction, I would like to have a dimmer for every light of the house with a physical dimmer. Ideally, I would also add in some pieces a colored led light. 
Could you give me advices to buy a supported dimmer to install on the wall, that will works correctly wiht Z2M ?
Thanks!

**Comment by tom-sherman** [comment]
You should be able to find a suitable device in the list of supported devices https://www.zigbee2mqtt.io/information/supported_devices.html

**Reply by lakano** [reply]
Thanks for the answer. 
Obviously this was my first step, but I don't found any precisions about colored led light, and if this fully working with dimming.
If someone have a physical dimmer 100% compatible Z2M with possibility to put colored light, I can buy the same brand/model :-)

**Reply by tom-sherman** [reply]
The bulb will specify if it's supports dimming in the product specification, you can verify this in the Z2M docs by looking to see if they support the "brightness" property.

I'm not aware of any off-the-shelf switches that allow you to change both the color and the brightness though.
